src/assets/css/librarybrowser.css

```diff
@@ -1122,3 +1122,22 @@ div:not(.sectionTitleContainer-cards) > .sectionTitle-cards {
 .itemsViewSettingsContainer > .button-flat {
     margin: 0;
 }
+
+.detail-clamp-text {
+    overflow: hidden;
+    display: -webkit-box;
+    -webkit-line-clamp: 12;
+    -webkit-box-orient: vertical;
+
+    /* Fallback for older browsers: line-height * number of lines */
+    max-height: calc(1.35 * 12em);
+}
+
+@media all and (min-width: 40em) {
+    .detail-clamp-text {
+        -webkit-line-clamp: 6;
+
+        /* Fallback for older browsers: line-height * number of lines */
+        max-height: calc(1.35 * 6em);
```

Comment:
It shouldn't need a fallback, as it's supported by everything we target: https://caniuse.com/#search=line-clamp (See the Browserslist key in package;json)